docker/IARA_ecs.sh

```diff
@@ -132,6 +137,27 @@ function save_iara_log() {
     echo "Completed."
 }
 
+function wait_for_s3_file () {
+    local timeout=60
+    local interval=2
+    local elapsed=0
+
+    echo "Aguardando o arquivo da rodada anterior (timeout: ${timeout}s)..."
+
+    while ! aws s3 ls s3://$S3_BUCKET/$IARA_FOLDER/$IARA_CASE/game_round_$IARA_GAME_ROUND/game_inputs.zip > /dev/null 2>&1; do
+        if [ "$elapsed" -ge "$timeout" ]; then
+            echo "Timeout: Arquivo não encontrado"
+            return 1
+        fi
+        echo "Arquivo não encontrado. Aguardando..."
+        sleep $interval
+        elapsed=$((elapsed + 5))
+    done
+
+    echo "Arquivo encontrado."
+    return 0
+}
+
```

Comment:
We should either rename the function to `wait_for_game_inputs_file` or pass the file path in s3 as a parameter.

Remember to translate the messages to english